cmake/templates/HPXMacros.cmake.in

```diff
@@ -95,25 +95,3 @@ function(hpx_check_allocator_compatibility)
       "informational purposes to dependent projects and should not be changed.")
   endif()
 endfunction()
-
-# Check we are using the same build type as HPX
-function(hpx_check_cmake_build_type)
-  get_property(_GENERATOR_IS_MULTI_CONFIG GLOBAL PROPERTY GENERATOR_IS_MULTI_CONFIG)
-  if (NOT _GENERATOR_IS_MULTI_CONFIG AND
-      NOT "${HPX_BUILD_TYPE}" STREQUAL "${CMAKE_BUILD_TYPE}" AND
```

Comment:
@hkaiser, @K-ballo, we used to skip checking that build types match for Visual Studio projects anyway. Is there anything we need to or can do to check that, or is that handled somehow automatically by Visual Studio?

Reply:
@msimberg I'm not sure we can do anything about it at configuration time as the generated VS projects are multi-target. The mismatch should be caught by the versioning symbols we generate, though (when building applications).